ajax/libs/angular-imgcache.js/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "angular-imgcache.js",
+  "version": "0.1.0",
+  "description": "Simple imgcache.js wrapper for AngularJS, can be user with Ionic/Cordova/Phonegap.",
```

Comment:
Is `user` a typo here? Suppose it's `used`?

Reply:
@extend1994 I used author`s description, but I thing you are right, I will update it